java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Add Content Project CRUD operations and expose them via XMLRPC
```

Comment:
I would mention `Environment` as well:

`- Add Content Project and Environment CRUD operations and expose them via XMLRPC`

Reply:
True! Fixing.